packages/next/src/server/base-server.ts

```diff
@@ -2316,7 +2316,7 @@ export default abstract class Server<ServerOptions extends Options = Options> {
           )
         } else if (isAppPageRouteModule(routeModule)) {
           if (
-            !opts.experimental.ppr &&
+            !renderOpts.experimental.ppr &&
```

Comment:
curious why this is changing?

Reply:
The rest of the render code inspects the `renderOpts` object which is the merged `opts` + other spreads. This just brings it in-line to be accessing the same instance rather than this one which was technically accessing a higher instance. In an ideal world, the construction of `renderOpts` would occur out of scope of this function such that it just gets the "complete" version.